data/coalition/coalition missions.txt

```diff
@@ -1393,3 +1393,65 @@ mission "Arachi Cartoonist 2"
+			branch navy
+				has "license: Navy"
+			choice
+				`	"No, I'm not with the Navy. You could've just asked without slamming me to the ground."`
+					goto notnavy
+				`	"Is that why you attacked me?"`
+					goto detained
+			label navy
+			choice
+				`	"I am with the Navy, but I didn't come here looking for you."`
+				`	"Is that why you attacked me?"`
+					goto detained
+			`	"And I'm supposed to believe that crap?" He lashes out, trying to knock the Arach off of him, but the agent only tightens the grip. She opens her mandibles, getting them close to the Alpha's neck. "Alright, alright, I get it!" the Alpha shouts, stopping his attempts to break free.`
+				goto detained
+			label notnavy
+			`	He stares at you for a moment, as if unsure if you're telling the truth. "What the hell are you doing here then? Did you just stumble onto this place?" He stops struggling, but the Arach agent doesn't loosen her grip.`
+			label detained
+			`	The Saryd walks closer to the Alpha, stomping his hoofs on the ground with each step. "Whatever your motivation was, a transgression, you have committed. Punished accordingly, you shall be." He says. "Kindly follow us to the detention facility, would you, Captain?" He asks you, but doesn't wait for an answer, as more agents arrive to escort you and the Alpha to the "detention facility."`
+			`	It's a short walk, and you end up in a building similar to a police station. The Alpha is brought before a Kimek that you guess is in charge of the facility. He asks you to give a testimony of what happened, with the Heliarch agents confirming what you say. "Your first offense in a long time this is, Alitis," the Kimek says, after checking a computer for the Alpha's data. "Three years in seclusion, your sentence is."`
+			`	After the short verdict, which seems to lack much of the due process you would expect in human space, the agents bring both you and the Alpha to a lower level of the building. This floor has several empty cells kept shut by thick, tough walls of some transparent material. While the cell block does appear to go further beyond this section and around a corner, the first few cells appear to be directly opposite a modest medical space, presumably used mostly for treating any injured prisoners without leaving the secure level. At present though, all of the cells appear to be empty, and they simply lock him in the second one where accessing his cell would not obstruct any other doorways. The agents then ask that you wait a few minutes for a medic to arrive so that they can take care of your bruises.`
```

Comment:
```suggestion
			`	After the short verdict, which seems to lack much of the due process you would expect in human space, the agents bring both you and the Alpha to a lower level of the building. This floor has several empty cells kept shut by thick, tough walls of some transparent material. While the cell block does appear to go further beyond this section and around a corner, the first few cells appear to be directly opposite a modest medical space, presumably used mostly for treating any injured prisoners without leaving the secure level. At present though, all of the cells appear to be empty, and the Heliarchs simply lock him in the second cell, where accessing his would not obstruct any other doorways. The agents then ask that you wait a few minutes for a medic to arrive so that they can take care of your bruises.`
```